spotlight-client/src/withRouteSync/withRouteSync.tsx

```diff
@@ -64,7 +64,9 @@ const withRouteSync = <Props extends RouteComponentProps & RouteParams>(
     // eslint-disable-next-line react-hooks/exhaustive-deps
     useEffect(
       action("sync route params", () => {
-        tenantStore.currentTenantId = normalizedProps.tenantId ?? undefined;
+        if (!tenantStore.locked) {
```

Comment:
Without this condition, we run through this action twice and end up setting currentTenantId to our `normalizedProps.tenantId` which is not what we want. I don't see this occurring on production however?